CMakeLists.txt

```diff
@@ -42,6 +41,12 @@ else()
   message("Skipping python libraries.")
 endif()
 
+if(NOT APPLE)
```

Comment:
Why if not Apple? We don't need lapack on macOS?

Reply:
Yes, I believe so. It seems we never use BLAS or LAPACK directly. It's just a dependency of armadillo. [It looks like](https://github.com/apple/turicreate/blob/master/src/external/armadillo/CMakeLists.txt#L162) on macOS armadillo uses the Accelerate Framework instead of the BLAS and LAPACK.

@hoytak or @srikris - can you confirm?

Reply:
I see -- sounds right to me.